apps/web/lib/emails/templates/attendee-scheduled-email.ts

```diff
@@ -286,13 +298,13 @@ ${getRichDescription(this.calEvent)}
   }
 
   protected getAdditionalNotes(): string {
-    if (!this.calEvent.description) return "";
+    if (!this.calEvent.additionalNotes) return "";
```

Comment:
Is this intended? @sean-brydon

Reply:
Yes @zomars 

calEvent.description was previously set to the addiitonalNotes for some reason. Talked with peer and we have renamed that field in the type to be additionalNotes - and created a new one called description.

AditionalNotes: Holds the old description (additional notes)
Description: Holds the event description